src/semigroup/object.js

```diff
@@ -13,6 +12,27 @@ Semigroup.instance(Object, {
   }
 });
 
+/**
+ * Analogue of Object.assign(). Copies properties from one or more source objects to
+ * a target object. Existing keys on the target object will be overwritten. 
+ * Modified to support copying Symbols.
+ */
```

Comment:
Should we add a comment in here this exists because React Native does weird things?

Reply:
Yeah, good idea.